pug/components/cart.pug

```diff
@@ -0,0 +1,39 @@
+mixin cartTable(itemList)
+  table.cart-table
+    tr 
+      th hình ảnh
+      th Tên sản phẩm
+      th Đơn giá
+      th Số lượng
+      th Thành tiền
+      th Xóa
+    each item in itemList 
+      +cartItem(item)
+      
+mixin cartItem(item)
+  tr 
+    td.cart-table__img
+      img(src=item.imgSrc)
+    td(class="cart-table__title")=item.title
+    td=`${item.curPrice}.000 đ`
```

Comment:
Hoàn toàn có thể viết function để xử lý nhé, k nhất thiết phải làm thủ công như này
```suggestion
mixin cartItem(item)
  -
     function convertMoney(money) {
         return money.toFixed(0).toString().split(/(?=(?:\d{3})+(?:\.|$))/g).join('.') + ' đ';
     }
  tr 
    td.cart-table__img
      img(src=item.imgSrc)
    td(class="cart-table__title")=item.title
    td=`${convertMoney(curPrice)}`
```